runtime/interface.h

```diff
@@ -11,6 +11,7 @@
 #include "runtime/kphp_core.h"
 #include "runtime/optional.h"
 #include "server/php-query-data.h"
+#include "server/web-server-stats.h"
```

Comment:
It's needed for include `f$get_webserver_stats()`

Reply:
Oh.. I see, but it's not very good option to do like this) Because it's extremely implicit and unclear for IDE, people and compiler
Imagine that someone will see that this include is unused and just remove it. It will implicitly break everything
Let's move `f$get_webserver_stats()` definition/declaration to interface.h/cpp or somewhere else in runtime folder